ui/mods/msu/mod_settings/enum_setting.js

```diff
@@ -0,0 +1,59 @@
+var EnumSetting = function (_mod, _page, _setting, _parentDiv)
+{
+	var self = this;
+	this.setting = _setting;
+	this.idx = _setting.array.indexOf(_setting.value);
+	if (this.idx == -1)
+	{
+		console.error("EnumSetting Error");
+	}
+
+	this.layout = $('<div class="enum-container"/>');
+	_parentDiv.append(this.layout);
+
+	this.title = $('<div class="title title-font-big font-bold font-color-title">' + _setting.name + '</div>');
+	this.layout.append(this.title);
+
+	this.button = this.layout.createTextButton(_setting.value, function ()
+	{
+		self.cycle(true);
+	}, 'enum-button', 4);
+
+	this.button.mousedown(function (event)
+	{
+		if (event.which === 3)
```

Comment:
I think this can be merged into the eventlistener I defined with the button

Reply:
Yes

Reply:
actually no it can't right now, as createTextButton doesn't pass the event

Reply:
Out of my scope